tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -132,46 +119,6 @@ Then('no message should be displayed on the webUI', function () {
   return client.page.webPage().expect.element('@message').to.not.be.present
 })
 
-Then(
-  'as {string} the content of {string} should be the same as the content of local file {string}',
-  async function (userId, remoteFile, localFile) {
-    const fullPathOfLocalFile = path.join(client.globals.filesForUpload, localFile)
-    const body = await webdavHelper.download(userId, remoteFile)
-
-    assertContentOfLocalFileIs(fullPathOfLocalFile, body)
-
-    return this
-  }
-)
-
-Then(
-  'as {string} the content of {string} should not be the same as the content of local file {string}',
-  async function (userId, remoteFile, localFile) {
-    const fullPathOfLocalFile = path.join(client.globals.filesForUpload, localFile)
-    const body = await webdavHelper.download(userId, remoteFile)
-
-    assertContentOfLocalFileIsNot(fullPathOfLocalFile, body)
-  }
-)
-
-const assertContentOfLocalFileIs = function (fullPathOfLocalFile, actualContent) {
-  const expectedContent = fs.readFileSync(fullPathOfLocalFile, { encoding: 'utf-8' })
-  return assert.strictEqual(
-    actualContent,
-    expectedContent,
-    'asserting content of local file "' + fullPathOfLocalFile + '"'
-  )
-}
-
-const assertContentOfLocalFileIsNot = function (fullPathOfLocalFile, actualContent) {
-  const expectedContent = fs.readFileSync(fullPathOfLocalFile, { encoding: 'utf-8' })
-  return assert.notStrictEqual(
-    actualContent,
-    expectedContent,
-    'asserting content of local file "' + fullPathOfLocalFile + '"'
-  )
-}
-
 Given(
   'the setting {string} of app {string} has been set to {string}',
```

Comment:
These steps and some below are present in middleware. Why not use middleware's?

Reply:
refactoring for this step is already done in https://github.com/owncloud/web/pull/6188 (which is blocked until this PR), so avoided here